lib/LANraragi/Utils/Database.pm

```diff
@@ -226,14 +227,17 @@ sub delete_archive($id) {
     if ( -e $filename ) {
         unlink $filename;
 
-        my $use_jxl = LANraragi::Model::Config->get_jxlthumbpages;
-        my $format = $use_jxl ? 'jxl' : 'jpg';
         my $thumbdir  = LANraragi::Model::Config->get_thumbdir;
         my $subfolder = substr( $id, 0, 2 );
-        my $thumbname = "$thumbdir/$subfolder/$id.$format";
-        # this should probablydelete the thumbpages folder too?
 
-        unlink $thumbname;
+        my $jpg_thumbname = "$thumbdir/$subfolder/$id.jpg";
+        unlink $jpg_thumbname;
+
+        my $jxl_thumbname = "$thumbdir/$subfolder/$id.jxl";
+        unlink $jxl_thumbname;
+
+        # Delete the thumbpages folder
+        remove_tree("$thumbdir/$subfolder/$id/");
```

Comment:
i hope this is fine, before LRR did not remove the thumbpages folder when deleting an archive

Reply:
Yeah, I believe this was an oversight on my part back when adding thumbpages. Thanks!